docker/mobile-testkit/Dockerfile

```diff
@@ -25,20 +22,27 @@ RUN cd /tmp/ && \
     cd libcouchbase-2.5.6_centos7_x86_64 && \
     rpm -ivh libcouchbase-devel-2.5.6-1.el7.centos.x86_64.rpm \
```

Comment:
It would be great to pull the latest versions of libcouchbase* rpms instead of a fixed version